src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$gap: 1px;
+$cell-size: 100px;
+$calendsar: 7 * ($cell-size + $gap);
```

Comment:
what is `calendsar`? be more specific with your variable names, use something like `calendar-max-width`